en/homework/README.md

```diff
@@ -18,7 +18,7 @@ Remember the chapter about querysets? We created a view `post_list` that display
 
 Time to do something similar, but for draft posts.
 
-Let's add a link in `blog/templates/blog/base.html` in the header. We don't want to show our list of drafts to everybody, so we'll put it inside the `{% if user.is_authenticated %}` check, right after the button for adding new posts.
+Let's add a link in the `<div class="page-header">` section of `blog/templates/blog/base.html`. We don't want to show our list of drafts to everybody, so we'll put it inside the `{% if user.is_authenticated %}` check, right after the button for adding new posts.
```

Comment:
I would just say `Let's add a link in the page-header section of ` instead.

Reply:
Would you like to commit this change to this PR's branch, @amakarudze?